doc/Windows.md

```diff
@@ -0,0 +1,28 @@
+## Local development with Windows in WSL 2 environment
+First you have to make sure, you are running Docker desktop in windows. See [Docker Desktop](https://www.docker.com/products/docker-desktop/) for more information.
+
+ 
+### 1. Build with the docker-compose command
+```
```

Comment:
Do we need the 1. ?

Reply:
Yes, this is telling the number of steps